packages/react/src/experimental/Modals/OneModalHeader/OneModalHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import { ButtonInternal } from "@/components/Actions/Button/internal"
+import {
+  DropdownInternal,
+  DropdownInternalProps,
+} from "@/experimental/Navigation/Dropdown/internal"
+import CrossIcon from "@/icons/app/Cross"
+
+export type OneModalHeaderProps = {
+  title: string
+  dropdownItems?: DropdownInternalProps["items"]
```

Comment:
we usually call those things `otherActions` to convey the semantics, not implementation details